doc/developers/contributing.rst

```diff
@@ -1550,6 +1550,10 @@ poor_score
 multioutput_only
     whether estimator supports only multi-output classification or regression.
 
+binary_only
+    whether estimator supports binary classification but lacks multi-class
+    classification support.
```

Comment:
I guess just following the tag logic this is true since an estimator has to "opt in" to multilabel. I don't think there are any multilabel tests though :confused: